src/elements/public/Table/index.ts

```diff
@@ -1,3 +1,4 @@
+import '../../internal/InternalSandbox/index';
```

Comment:
@pheekus This one too?

Reply:
@brettflorio yes these register an internal sandbox element with a shadow dom where templates are rendered.

Reply:
Ok great. Just checking :)